src/pages/home/components/ActionBar/ActionBar.tsx

```diff
@@ -1,21 +1,28 @@
 import { ReactNode } from 'react';
 
+import { useAccountStore } from '~stores/useAccountStore/useAccountStore';
+
 import Button from '~components/Button/Button';
 import Icon from '~components/Icon/Icon';
 
 import scss from './ActionBar.module.scss';
 
 function ActionBar(): ReactNode {
+  const { accounts } = useAccountStore();
+
   return (
     <div className={scss.container}>
       <div className={scss.composeContainer}>
         <Icon className={scss.closeIcon} icon="close" size={14} />
         <p className={scss.composeLabel}>Compose</p>
       </div>
-      <button className={scss.navigationIconContainer}>
-        <Icon className={scss.arrowIcon} icon="arrow-right" size={12} />
-      </button>
-
+      <Button
+        circle
+        className={scss.navigationIconContainer}
+        color="secondary"
+        disabled={accounts.length === 0}
```

Comment:
Usa isEmpty do lodash para seguir a padronização do projeto
```suggestion
        disabled={isEmpty(accounts)}
```